configure.py

```diff
@@ -1517,7 +1517,7 @@ def main():
     else:
       set_trisycl_include_dir(environ_cp)
 
-  set_action_env_var(environ_cp, 'TF_NEED_ROCM', 'ROCm', False)
```

Comment:
since we have `configure_input` now, can we leave this value default to `False`?

Reply:
yes, will update